Blackstar/BlackstarClient.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Blackstar
+{
+    public class BlackstarClient
+    {
+        readonly HttpClient _client;
+        readonly BlackstarServerRoutes _routes;
+
+        public BlackstarClient(string serverUrl)
+        {
+            if (serverUrl == null) throw new ArgumentNullException(nameof(serverUrl));
+            _routes = new BlackstarServerRoutes(new Uri(serverUrl));
+            _client = new HttpClient();
+        }
+
+        public async Task<ContentChunk[]> GetAllAsync()
```

Comment:
`*Async` naming? Not sure if people still use it; thought it might be good to follow the `HttpClient` analogy. Ctrl+R'd it anyway :-)